scripts/test_examples.sh

```diff
@@ -201,43 +212,255 @@ stopKafka
+testOutput "${cmd}" "${expected}" "${unexpected}"
+
+
+# Let us test - in example one new message generated
+expected=("offset arg not set, using default value
+Offset: 0, message count: 1
+All plugins initialized successfully
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Sending 1 sync Kafka notifications (protobuf) ...
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '10', key: 'proto-key', 
+Sending 1 async Kafka notifications (protobuf) ...
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '10', key: 'async-proto-key'
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '10', key: 'async-proto-key', 
+Sync watcher closed
+Async watcher closed
+")
+
+unexpected=("Error while stopping watcher
+")
+
+# this does not work: cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -messageCount=\"1\""
+cmd='examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf --messageCount 1'
+testOutput "${cmd}" "${expected}" "${unexpected}"
+
+# Let us test - in example one new message generated - with offset 11 for both topics and we display all messages from offset 8
+expected=("Offset: 8, message count: 1
+All plugins initialized successfully
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '8', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '10', key: 'async-proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '8', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '10', key: 'proto-key', 
+Sending 1 sync Kafka notifications (protobuf) ...
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '11', key: 'proto-key', 
+Sending 1 async Kafka notifications (protobuf) ...
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '11', key: 'async-proto-key'
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '11', key: 'async-proto-key', 
+Sync watcher closed
+Async watcher closed
+")
+
+unexpected=("Error while stopping watcher
```

Comment:
add here Received async Kafka Message, topic 'example-async-topic', partition '2', offset '7', key: 'async-proto-key',
since we are watching offset 8, we do not want to see the offset before it.